climpred/prediction.py

```diff
@@ -838,12 +839,35 @@ def compute_persistence_pm(ds, control, nlags, metric='pearson_r',
             'rmse',
             'mse',
             'mae'""")
+
+    init_years = ds['initialization'].values
+    if isinstance(ds.time.values[0], cftime._cftime.DatetimeProlepticGregorian) or isinstance(ds.time.values[0], np.datetime64):
```

Comment:
Not necessary now, but if we find in other functions that we need to deal with datetimes in a similar way, we can move this chunk of code into its own function.

Reply:
but isnt it also a design question: where to deal with time when it is datetime? for lagged persistence we need to do something inside the function...